app/api/cron/dune/program-calls/route.ts

```diff
@@ -0,0 +1,67 @@
+import { headers } from 'next/headers';
+import { db } from '@/src/db/drizzle';
+import { program_call_stats } from '@/src/db/schema';
+import { NextResponse } from 'next/server';
+import { DuneClient, RunQueryArgs } from '@duneanalytics/client-sdk';
+import { Cluster } from '@utils/cluster';
+import { fetchProgramMetadataIdl, programNameFromIdl } from '@/app/components/instruction/codama/getProgramMetadataIdl';
+import { PROGRAM_INFO_BY_ID } from '@/app/utils/programs';
+
+const { DUNE_API_KEY, DUNE_PROGRAM_CALLS_MV_ID } = process.env;
```

Comment:
Let's handle envs in the same pattern. `process.env.CRON_SECRET` on the 14th line is used without destructive assignment. I'd prefer these two to be used the same way